data/migrations/20190817144837_Projects.js

```diff
@@ -0,0 +1,14 @@
+exports.up = function(knex, Promise) {
+    return knex.schema.createTable("projects", tbl => {
+        tbl.increments();
+        tbl.string("name")
+            .notNullable()
+            .unique();
+        tbl.string("description").nullable();
+        tbl.boolean("complete").defaultTo(false)
+    });
```

Comment:
what if a user on the frontend were to want to say that was something completed before entering it into their database. Would it return a truthy value if they clicked that it was completed? Or would this migration table override it to false?